pallets/ddc-payouts/src/lib.rs

```diff
@@ -198,14 +198,22 @@ pub mod pallet {
 			cluster_id: ClusterId,
 			era: DdcEra,
 		},
-		Rewarded {
+		ProvidersRewarded {
```

Comment:
don't change the names - these events are already captured by DAC

Reply:
I think the naming doesn't play a significant role here. The index, the order of fields, and the types are what matters for successful event parsing.

Reply:
Anyway, returned as it was